### DISCUSSION TITLE
Missing an item in Iped search

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2168] [Q&A]
After indexed image of a Samsung SM-G780G, obtained by cellebrite, which generated the files below.

17/08/2023  12:26     4.068.589.593 EvidenceCollection_2023-08-17_Report.ufdr
17/08/2023  12:06     4.928.307.200 EvidenceCollection_2023-08-17_Report.z01
17/08/2023  12:07     4.928.307.200 EvidenceCollection_2023-08-17_Report.z02
17/08/2023  12:07     4.928.307.200 EvidenceCollection_2023-08-17_Report.z03
17/08/2023  12:08     4.928.307.200 EvidenceCollection_2023-08-17_Report.z04
17/08/2023  12:08     4.928.307.200 EvidenceCollection_2023-08-17_Report.z05
17/08/2023  12:09     4.928.307.200 EvidenceCollection_2023-08-17_Report.z06
17/08/2023  12:09     4.928.307.200 EvidenceCollection_2023-08-17_Report.z07
17/08/2023  12:10     4.928.307.200 EvidenceCollection_2023-08-17_Report.z08
17/08/2023  12:11     4.928.307.200 EvidenceCollection_2023-08-17_Report.z09
17/08/2023  12:11     4.928.307.200 EvidenceCollection_2023-08-17_Report.z10
17/08/2023  12:12     4.928.307.200 EvidenceCollection_2023-08-17_Report.z11
17/08/2023  12:12     4.928.307.200 EvidenceCollection_2023-08-17_Report.z12
17/08/2023  12:13     4.928.307.200 EvidenceCollection_2023-08-17_Report.z13
17/08/2023  12:14     4.928.307.200 EvidenceCollection_2023-08-17_Report.z14
17/08/2023  12:24     4.928.307.200 EvidenceCollection_2023-08-17_Report.z15

But when I looked at IPEDSearch I noticed that it was missing, the item EvidenceCollection_2023-08-17_Report.z01.

![image](https://github.com/sepinf-inc/IPED/assets/381594/ce4ffa89-c586-4295-a2a1-fb70fe741065)

is it correct?

paulo

**Comment by lfcnassif** [comment]
Hi @paulobreim. It's not, you should process the UFDR file, even if it is segmented, not Its parent folder, as warned on #1272.

**Reply by paulobreim** [reply]
Hummm yes, I forgot it.
tks
